CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Changelog
 
 ## v0.5.3
+  * Enchancements
+    * We now collect operating system-level messages from `/dev/log` and
```

Comment:
Could you move this kind of documentation to the README.md? In general, we try to keep the Changelog to one liners that discuss the feature. Since syslog and klog messages were getting dropped the previous releases, I think that it's safe to say that this is a new feature and that we forward syslog and klog messages to the Elixir `Logger` now.